### PR TITLE
Create maya2018.ebuild

### PR DESCRIPTION
**Pull request by tux1989**
This is EBUILD for Autodesk Maya 2018

### CODE REVIEW

File: media-gfx/maya2018/maya2018.ebuild [hunk 1]
```diff
@@ -0,0 +1,143 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header:  Maya-2018 for amd64$
```
Comment (Zlogene) [inline media-gfx/maya2018/maya2018.ebuild]: it is obsolete

File: media-gfx/maya2018/maya2018.ebuild [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+EAPI="4"
```
Comment (Zlogene) [inline media-gfx/maya2018/maya2018.ebuild]: too old eapi

File: media-gfx/maya2018/maya2018.ebuild [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+inherit rpm eutils
+
+DESCRIPTION="Autodesk's Maya. Commercial modeling and animation package"
```
Comment (Zlogene) [inline media-gfx/maya2018/maya2018.ebuild]: `DESCRIPTION` should not contain a package name

File: media-gfx/maya2018/maya2018.ebuild [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+RESTRICT="mirror nouserpriv"
+
+# Needed for install
+DEPEND="app-arch/rpm2targz app-arch/tar"
+
+# MayaPy needs at least this to work:
+RDEPEND="app-shells/tcsh
+	media-libs/libpng:1.2
+	dev-lang/python
```
Comment (Zlogene) [inline media-gfx/maya2018/maya2018.ebuild]: not needed

File: media-gfx/maya2018/maya2018.ebuild [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+	x11-libs/libXinerama
+	x11-libs/libXrender
+	media-libs/fontconfig
+	media-libs/glu
+	amd64? ( app-emulation/emul-linux-x86-gtklibs )
```
Comment (Zlogene) [inline media-gfx/maya2018/maya2018.ebuild]: long ago gone, there is a native multilib support

File: media-gfx/maya2018/maya2018.ebuild [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+	virtual/jpeg:62
+"
+
+RDEPEND="${RDEPEND}
+	=media-gfx/adlmapps-14
```
Comment (Zlogene) [inline media-gfx/maya2018/maya2018.ebuild]: why do you need specific version?

File: media-gfx/maya2018/maya2018.ebuild [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+	=media-gfx/adlmflexnet-14
```
Comment (Zlogene) [inline media-gfx/maya2018/maya2018.ebuild]: here too?

File: media-gfx/maya2018/maya2018.ebuild [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+"
+
+# The ./setup program needs these two libs to work
+RDEPEND="${RDEPEND} x11-libs/libXrandr x11-libs/libXft"
```
Comment (Zlogene) [inline media-gfx/maya2018/maya2018.ebuild]: one dependency atom per string, everywhere

File: media-gfx/maya2018/maya2018.ebuild [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+pkg_postrm() {
+	einfo "Be aware that Maya likes to leave files hanging, like creating totaly unpredicted files"
+	einfo "And you know what that means, even now there's crap in your system left there by maya..."
+	einfo
+	einfo "These directories are probably present in your system:"
+	einfo " * /var/opt/Autodesk/"
+	einfo
+	einfo "And maybe more, who knows?"
+}
+© 2018 GitHub, Inc.
```
Comment (Zlogene) [inline media-gfx/maya2018/maya2018.ebuild]: take this garbage out

File: media-gfx/maya2018/maya2018.ebuild [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+
+LICENSE="maya-2018"
+SLOT="2018"
+KEYWORDS="~amd64"
+IUSE="bundled-libs openmotif"
+
+RESTRICT="mirror nouserpriv"
+
+# Needed for install
+DEPEND="app-arch/rpm2targz app-arch/tar"
```
Comment (Zlogene) [inline media-gfx/maya2018/maya2018.ebuild]: do not depend on tar

File: media-gfx/maya2018/maya2018.ebuild [hunk 11]
```diff
@@ -0,0 +1,143 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header:  Maya-2018 for amd64$
+EAPI="4"
+
+inherit rpm eutils
+
+DESCRIPTION="Autodesk's Maya. Commercial modeling and animation package"
+HOMEPAGE="http://usa.autodesk.com/maya/"
```
Comment (Zlogene) [inline media-gfx/maya2018/maya2018.ebuild]: https

File: media-gfx/maya2018/maya2018.ebuild [hunk 12]
```diff
@@ -0,0 +1,143 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header:  Maya-2018 for amd64$
+EAPI="4"
+
+inherit rpm eutils
```
Comment (Zlogene) [inline media-gfx/maya2018/maya2018.ebuild]: why do you need eutils?

File: media-gfx/maya2018/maya2018.ebuild [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+
+
+# Stuff I'm not sure about
+RDEPEND="${RDEPEND}
+	x11-libs/libxcb app-admin/gamin dev-libs/libgamin
+	media-libs/libquicktime media-libs/audiofile
+	sys-libs/e2fsprogs-libs media-libs/openal
+	amd64? (
+		!bundled-libs? ( x11-libs/libXpm x11-libs/libXmu x11-libs/libXt x11-libs/libXp x11-libs/libXi x11-libs/libXext x11-libs/libX11 x11-libs/libXau x11-libs/libxcb )
+		bundled-libs? ( app-emulation/emul-linux-x86-xlibs app-emulation/emul-linux-x86-baselibs app-emulation/emul-linux-x86-qtlibs )
```
Comment (Zlogene) [inline media-gfx/maya2018/maya2018.ebuild]: again, no more relevant

**Comment by mgorny** [comment]
I'm sorry but we don't accept ebuilds made through GitHub web interface. The ebuild submitted has numerous issues to the point of not even being a valid ebuild, and you clearly haven't tested it. I'm going to close this and the following PRs not to waste our CI's computing power on something this obviously broken.

If you'd like to contribute to Gentoo, please read through our documentation in order to write correct ebuilds that conform to our current standards, and use associated tooling to test those ebuilds. Do not submit ebuilds which you haven't tested.

Furthermore, if you haven't written those files yourself but instead copied them from somewhere, please ensure that you are legally allowed to submit them to Gentoo and that you attribute the original author(s).

**Pull request closed without merging**